hpc_plugin/tasks.py

```diff
@@ -327,19 +356,33 @@ def deploy_job(script,
         call = "./" + name
         for dinput in inputs:
             call += ' ' + dinput
-        _, exit_code = client.execute_shell_command(
+        # ctx.logger.info('deploy_job: call=%s'%call)
+        output, exit_code = client.execute_shell_command(
             call,
             workdir=workdir,
             wait_result=True)
+        if output:
+            ctx.logger.info('deploy_job: output=%s'%output)
+        
+        # get output and add log to logger
+        cmd_out, cmd_code = client.execute_shell_command(
+            'cat ' + name + '.log',
+            workdir=workdir,
+            wait_result=True)
+        if cmd_out:
+            ctx.logger.info('deploy_job: cmd_out=%s'%cmd_out)
```

Comment:
If the log is big, I'm afraid it will degrade the logs user experience.
What correspond to "name" in this case?
Is this the application log?